fe/src/project/AspNetCore/areas/News/components/NewsCatalog/NewsCatalog.scss

```diff
@@ -0,0 +1,90 @@
+.news-catalog {
+
+    .image img {
+        border-radius: 0.5rem;
+    }
+
+    .hero-news {
+        cursor: pointer;
+    }
+
+    .data-text {
+        margin-left: 0.5rem;
+    }
+
+    .news-data {
+        display: flex;
+        align-items: center;
+        margin: 0.5rem 0;
+
+        .data {
+            font-size: 0.8rem;
+            display: flex;
+            align-items: center;
+
+            svg {
+                fill: $primary;
+            }
+
+            &:first-child {
+                margin-right: 1rem;
+            }
+        }
+    }
+
+    .news-title {
+        font-size: 1.8rem;
+        font-weight: 700;
+        line-height: 2.2rem;
+    }
+
+    .news-description {
+        margin-top: 1.2rem;
+        display: -webkit-box;
+        -webkit-line-clamp: 2;
+        -webkit-box-orient: vertical;
+        overflow: hidden;
+    }
+
+    &__news {
+        margin-top: 1rem;
+
+        .card {
+            box-shadow: none;
+            cursor: pointer;
+
+            &:hover {
+                box-shadow: 0 .5em 1em -.125em rgba(10,10,10,.1),0 0 0 1px rgba(10,10,10,.02);
+            }
+        }
+
+        .media-content {
+            padding: 1rem;
+        }
+    }
+
+    &__categories {
+        display: flex;
+        flex-wrap: wrap;
+        margin-top: 1rem;
+
+        .category-tag {
```

Comment:
**suggestion:** let's remove it completely and use the standard material ui buttons. They already have all the styling.